CONTRIBUTING.md

```diff
@@ -1,22 +1,78 @@
-# Welcome!
+# Welcome!	
```

Comment:
Not sure why these weird whitespaces got added throughout the file.

Reply:
Fixed.